packages/arb-token-bridge-ui/src/util/TokenUtils.ts

```diff
@@ -235,3 +238,49 @@ export async function l1TokenIsDisabled({
   const erc20Bridger = await Erc20Bridger.fromProvider(l2Provider)
   return erc20Bridger.l1TokenIsDisabled(erc20L1Address, l1Provider)
 }
+
+type SanitizeTokenOptions = {
+  erc20L1Address?: string | null // token address on L1
+  chain: Chain // chain for which we want to retrieve the token name / symbol
+}
+
+// get the exact token symbol for a particular chain
+export function sanitizeTokenSymbol(
+  tokenSymbol: string,
+  options?: SanitizeTokenOptions
+) {
+  if (typeof options === 'undefined') {
+    return tokenSymbol.toUpperCase()
+  }
+
+  const isArbitrumOne = isNetwork(options.chain.id).isArbitrumOne
+
+  // only special case for USDC is Arbitrum One
+  if (options.erc20L1Address === CommonAddress.Mainnet.USDC && isArbitrumOne) {
+    return 'USDC.e'
+  }
+
+  return tokenSymbol.toUpperCase()
+}
+
+// get the exact token name for a particular chain
+export function sanitizeTokenName(
+  tokenName: string,
+  options?: SanitizeTokenOptions
```

Comment:
why would this be optional?
if it returns `tokenName`, might as well not call this method?
there shouldn't be a case where we don't pass in the erc20token address but need to pass in a token name and simply return the token name

Reply:
Yeah that's fair it probably shouldn't be optional

Reply:
Yeah, we don't call sanitize function without the options ever. So I now made it consistent in both functions and made it a required param.